sogo.spec

```diff
@@ -33,7 +33,7 @@ URL:          http://www.inverse.ca/contributions/sogo.html
 Group:        Productivity/Groupware
 Source:       https://github.com/inverse-inc/sogo/archive/SOGo-%{sogo_version}.tar.gz
 Prefix:       /usr
-AutoReqProv:  off
+AutoReqProv:  no
```

Comment:
yes thinking on it, it seems that probably yum considered that it was allowed to perform update because it was not set to `no`

What is the safe option, set to `yes` or completely remove this ?

Reply:
It a error left over from the upstream spec, ill say we correct it.

We **might** break installs who did **not** pull the epel packages in, but they are rare (only one?) and we should not live with an mistake forever for this.

again let's do it.

Reply:
I mean set it to **no**